mlflow/environment_variables.py

```diff
@@ -204,6 +204,32 @@ def get(self):
     "MLFLOW_DEFAULT_PREDICTION_DEVICE", str, None
 )
 
+#: Specifies to Huggingface whether to use the automatic device placement logic of
+# HuggingFace accelerate. If it's set to false, the low_cpu_mem_usage flag will not be
+# set to True and device_map will not be set to "auto".
+MLFLOW_HUGGINGFACE_DISABLE_ACCELERATE_FEATURES = _EnvironmentVariable(
```

Comment:
Did we want this to be a `_BooleanEnvironmentVariable` as well?

Reply:
Great point - will do in the next update (want to see the result of tests first)